bpf/lib/nat.h

```diff
@@ -1533,6 +1540,27 @@ snat_v6_needs_masquerade(struct __ctx_buff *ctx __maybe_unused,
 		}
 	}
 
+/* Check if the packet matches an egress NAT policy and so needs to be SNAT'ed.
+ *
+ * This check must happen before the IPV4_SNAT_EXCLUSION_DST_CIDR check below as
```

Comment:
```suggestion
 * This check must happen before the IPV6_SNAT_EXCLUSION_DST_CIDR check below as
```

Alternatively just drop the comment, when it's otherwise identical to the IPv4 path?